src/main/io/dji_osd_symbols.h

```diff
@@ -62,20 +62,41 @@
 #define DJI_SYM_HEADING_DIVIDED_LINE    0x1C
 #define DJI_SYM_HEADING_LINE            0x1D
 
-// AH Center screen Graphics
+// AH Center Screen Graphics
 #define DJI_SYM_CROSSHAIR_LEFT          0x72
 #define DJI_SYM_CROSSHAIR_CENTRE        0x73
 #define DJI_SYM_CROSSHAIR_RIGHT         0x74
-#define DJI_SYM_AH_RIGHT                0x02
-#define DJI_SYM_AH_LEFT                 0x03
+#define DJI_SYM_AH_RIGHT                0x03
+#define DJI_SYM_AH_LEFT                 0x02
 #define DJI_SYM_AH_DECORATION           0x13
 #define DJI_SYM_SMALL_CROSSHAIR         0x7E
 
+// Crosshair Styles
+#define DJI_SYM_AH_CH_TYPE3             0x00 // DJI_SYM_NONE
+#define DJI_SYM_AH_CH_TYPE3_1           0x7E // DJI_SYM_SMALL_CROSSHAIR
+#define DJI_SYM_AH_CH_TYPE3_2           0x00 // DJI_SYM_NONE
+#define DJI_SYM_AH_CH_TYPE4             0x2D // DJI_SYM_HYPHEN
+#define DJI_SYM_AH_CH_TYPE4_1           0x7E // DJI_SYM_SMALL_CROSSHAIR
+#define DJI_SYM_AH_CH_TYPE4_2           0x2D // DJI_SYM_HYPHEN
+#define DJI_SYM_AH_CH_TYPE5             0x17 // DJI_SYM_STICK_OVERLAY_HORIZONTAL
+#define DJI_SYM_AH_CH_TYPE5_1           0x7E // DJI_SYM_SMALL_CROSSHAIR
+#define DJI_SYM_AH_CH_TYPE5_2           0x17 // DJI_SYM_STICK_OVERLAY_HORIZONTAL
+#define DJI_SYM_AH_CH_TYPE6             0x00 // DJI_SYM_NONE
+#define DJI_SYM_AH_CH_TYPE6_1           0x09 // DJI_SYM_STICK_OVERLAY_SPRITE_MID
+#define DJI_SYM_AH_CH_TYPE6_2           0x00 // DJI_SYM_NONE
+#define DJI_SYM_AH_CH_TYPE7             0x78 // DJI_SYM_ARROW_SMALL_LEFT
+#define DJI_SYM_AH_CH_TYPE7_1           0x7E // DJI_SYM_SMALL_CROSSHAIR
+#define DJI_SYM_AH_CH_TYPE7_2           0x77 // DJI_SYM_ARROW_SMALL_RIGHT
+#define DJI_SYM_AH_CH_TYPE8             0x02 // DJI_SYM_AH_LEFT
+#define DJI_SYM_AH_CH_TYPE8_1           0x7E // DJI_SYM_SMALL_CROSSHAIR
+#define DJI_SYM_AH_CH_TYPE8_2           0x03 // DJI_SYM_AH_RIGHT
```

Comment:
You are defining characters that do not exist. This has been mentioned to you before. You need to stop doing this.